pkg/translate/output.go

```diff
@@ -111,41 +111,69 @@ func getBlockSize(domainVisited map[string]int, adminBlockSize string) string {
 	return strconv.Itoa(int(bs))
 }
 
-func toBlockTopology(wr io.Writer, root *topology.Vertex) error {
-	// traverse tree topology and when a node is reached, check within blockRoot for domain and print that domain.
+/*
+	func toBlockTopologyBFS(wr io.Writer, root *topology.Vertex) error {
+		// traverse tree topology and when a node is reached, check within blockRoot for domain and print that domain.
+		// keep a map of which domain has been printed
+		treeRoot := root.Vertices[topology.TopologyTree]
+		blockRoot := root.Vertices[topology.TopologyBlock]
+		visited := make(map[string]bool)
+		queue := []*topology.Vertex{treeRoot}
+		domainVisited := make(map[string]int)
+
+		if treeRoot != nil {
+			for len(queue) > 0 {
+				v := queue[0]
+				queue = queue[1:]
+
+				// sort the IDs
+				keys := sortVertices(v)
+				for _, key := range keys {
+					w := v.Vertices[key]
+					if len(w.Vertices) == 0 { // it's a leaf; don't add to queue
+						err := findBlock(wr, w.ID, blockRoot, domainVisited)
+						if err != nil {
+							return err
+						}
+					} else if !visited[w.ID] {
+						queue = append(queue, w)
+						visited[w.ID] = true
+					}
+				}
+			}
+		}
+		err := printDisconnectedBlocks(wr, blockRoot, domainVisited)
+		if err != nil {
+			return err
+		}
+
+		blockSize := ""
+		if _, exists := root.Metadata[topology.KeyBlockSizes]; exists {
+			blockSize = root.Metadata[topology.KeyBlockSizes]
+		}
+		blockSize = getBlockSize(domainVisited, blockSize)
+		_, err = wr.Write([]byte(fmt.Sprintf("BlockSizes=%s\n", blockSize)))
+		return err
+	}
+*/
+func toBlockTopologyDFS(wr io.Writer, root *topology.Vertex) error {
```

Comment:
are we planning to have other types of `toBlockTopology` beside DFS? If not, perhaps we could keep `toBlockTopology()`?

Reply:
No, we do not. Just for review purposes. I can rename it